frontend/src/components/common/button/Button.tsx

```diff
@@ -0,0 +1,104 @@
+import styled from '@emotion/styled';
+import React from 'react';
+
+export default MyButton;
```

Comment:
For consistency, let's stick with named imports. Instead of `export default MyButton`, add the `export` keyword in front of the function declaration.